src/directives/share-button/share-button.directive.ts

```diff
@@ -9,28 +9,37 @@ import {
 } from '@angular/core';
 
 import { ShareButtonsService } from '../../services/share-buttons.service';
-import { ShareArgs } from '../../helpers';
+import { ShareArgs, ShareProvider, Helper } from '../../helpers';
 
 @Directive({
-    selector: '[shareButton]'
+    selector: '[sbShareButton]'
```

Comment:
yeah  i prefixed it to keep consistency with the other inputs/outputs of the directive